.github/workflows/main.yml

```diff
@@ -258,6 +346,11 @@ jobs:
     - uses: actions/checkout@v4
     - run: rustup update --no-self-update stable && rustup default stable
     - run: rustup target add wasm32-unknown-unknown
+    - uses: Swatinem/rust-cache@v2
+      with:
+        key: build-examples
+        cache-targets: |
+          debug
```

Comment:
This improved the runtime of the `build_benchmarks` job from `3m 57s` to `50s` in the [last run](https://github.com/Systemcluster/wasm-bindgen/actions/runs/10272820093/usage).

Reply:
Lets add this in a dedicated PR please.
In any case, as long as we don't get the other jobs down in time this won't make a difference.

Reply:
In that follow-up, I would say just slap it on everything, because we are running into the open-source runner limit already.

Reply:
That might be on me, I'm sorry for that. I didn't consider that runs in forks count against the origin limits. 
I wish there was a better alternative to test the CI, `act` is a bit awkward for this repo (and in general). I guess next time I'll run the CI in a private reuplod when it comes up. I hope it's only short term?

Reply:
> I didn't consider that runs in forks count against the origin limits.

Wow ... that would be surprising ...

> I hope it's only short term?

Sorry, what is only short term?

Reply:
> Sorry, what is only short term?

Runner limits? I don't know which limits you meant.

> Wow ... that would be surprising ...

I'm trying to find a reference but I can't, I must have misremembered. I've stared at the CI for too long today I think. 😄

Reply:
> > Sorry, what is only short term?
> 
> Runner limits? I don't know which limits you meant.

Ah, the limit is basically how many runners can run at the same time, that's always present unless you pay.

Reply:
There's another limit that's usage hours per month, which would be a bit more uncomfortable to hit. Good if it's the other one.